source/fundamentals/authentication/scram.txt

```diff
@@ -0,0 +1,100 @@
+.. _csharp-authentication-scram:
+
+=====
+SCRAM
+=====
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: .NET, salt, default, code example
+
+Overview
+--------
+
+**Salted Challenge Response Authentication Mechanism (SCRAM)** is a family of
+authentication mechanisms that use a challenge-response mechanism to authenticate
+the user. SCRAM-SHA-256, which uses the SHA-256 algorithm to hash your password, is the
+default authentication mechanism in {+mdb-server+} version 4.0
+and later. SCRAM-SHA-1, which uses the SHA-1 algorithm instead, is the default
+authentication mechanism in {+mdb-server+} versions earlier than 4.0.
+
+You can use SCRAM to authenticate to MongoDB Atlas, MongoDB
+Enterprise Advanced, and MongoDB Community Edition.
+
+.. tip:: SCRAM Mechanisms
+
+   To learn more about the SCRAM family of authentication mechanisms, see
+   `RFC 5802 <https://tools.ietf.org/html/rfc5802>`__ and
+   :wikipedia:`Salted Challenge Response Authentication Mechanism <Salted_Challenge_Response_Authentication_Mechanism>`
+   on Wikipedia.
+
+   For more information about the MongoDB implementation of SCRAM, see
+   :manual:`SCRAM </core/security-scram>` in the {+mdb-server+} manual.
+
+Code Placeholders 
+~~~~~~~~~~~~~~~~~
+
+The code examples on this page use the following placeholders:
+
+- ``<db_username>``: The MongoDB username of the user to authenticate.
+- ``<db_password>``: The MongoDB password of the user to authenticate.
+- ``<hostname>``: The network address of your MongoDB deployment.
+- ``<port>``: The port number of your MongoDB deployment. If you omit this parameter,
+  the driver uses the default port number (``27017``). You don't need a port number
+  when connecting to a MongoDB Atlas cluster.
+- ``<authenticationDb>``: The MongoDB database that contains the user's authentication
+  data. If you omit this parameter, the driver uses the default value, ``admin``.
+
+To use the code examples on this page, replace these placeholders with your own values.
+
+Using SCRAM Authentication in Your Application
+----------------------------------------------
+
+To use SCRAM to authenticate your MongoDB user, specify your
+MongoDB credentials, but don't specify an authentication mechanism.
```

Comment:
Question for tech reviewer: Do we need to show how to explicitly specify SCRAM-1 or SCRAM-256? I assumed users would want to use the best one supported by their deployment.

Reply:
If the user omits `authMechanism` and the server doesn't support SCRAM-256, does the driver default to SCRAM-1?

I'm wondering if the user would ever need to explicitly specify SCRAM-1, or if the driver would automatically negotiate the most secure SCRAM protocol.